src/styles/blocks/button.scss

```diff
@@ -0,0 +1,24 @@
+.button {
+  display: block;
+  width: 280px;
+  height: 50px;
+  border-radius: 8px;
+  background-color: $colorBlue;
+  font-weight: 600;
+  font-size: 15px;
+  line-height: 50px;
+  text-align: center;
+  transition: all $transitionDuration;
```

Comment:
Try to avoid using 'all' for transition, because it has bad effect on browser, it's better to state every property that you want to have transition on